pandora-client-web/src/components/settings/accountSettings.tsx

```diff
@@ -10,15 +10,26 @@ import _, { uniq } from 'lodash';
 import { ColorInput } from '../common/colorInput/colorInput';
 import { useColorInput } from '../../common/useColorInput';
 import { useConfirmDialog } from '../dialog/dialog';
+import { useNavigate } from 'react-router-dom';
 
 export function AccountSettings(): ReactElement | null {
+	const navigate = useNavigate();
 	const account = useCurrentAccount();
 
 	if (!account)
 		return <>Not logged in</>;
 
 	return (
 		<>
+			<Button className='slim' onClick={ () => { // temporary button
```

Comment:
why temporary, when will it be removed/changed?

also it is probably better to add `TODO` in comments like these so we can search for them

Reply:
Good point, but I am not yet fully sure what to do with them in the future. Will discuss it with Jomshir, too.